receiver/windowseventlogreceiver/README.md

```diff
@@ -33,6 +33,7 @@ Tails and parses logs from windows event log API using the [opentelemetry-log-co
 | `retry_on_failure.initial_interval` | `1 second`   | Time to wait after the first failure before retrying.                                                                                                                                                                                          |
 | `retry_on_failure.max_interval`     | `30 seconds` | Upper bound on retry backoff interval. Once this value is reached the delay between consecutive retries will remain constant at the specified value.                                                                                           |
 | `retry_on_failure.max_elapsed_time` | `5 minutes`  | Maximum amount of time (including retries) spent trying to send a logs batch to a downstream consumer. Once this value is reached, the data is discarded. Retrying never stops if set to `0`.                                                  |
+| `remote`                            | []           | Remote configuration for connecting to multiple machine to collect logs. Each entry includes `credentials` (with `username`, `password`, and optional `domain`) and `servers` (a list of server addresses).                                    |
```

Comment:
Why are we connecting to multiple machines? Generally we would just use multiple instances of a receiver in order to pull from multiple points.

Reply:
Thank you for the feedback.

My initial approach was to support connecting to multiple machines using a single instance of the receiver, which allows for a single set of credentials to be used across multiple servers, or different credentials for different groups of servers. This was done to simplify the configuration for users who might want to manage multiple remote servers from a single receiver instance.

However, I understand that the standard practice is to use multiple instances of a receiver. I will refactor the code to use multiple instances of the receiver for each remote server. Having a single instance will simplify the code. 

I'll make the changes and update the PR accordingly.

Reply:
@djaglowski I applied the changes and the code looks simpler now 👍

Reply:
I wasn't aware of this pattern for remote machines @djaglowski thanks for pointing that out.